frontend/src/app/main/ui/workspace/libraries.cljs

```diff
@@ -152,7 +152,9 @@
         locale       (mf/deref i18n/locale)
         project      (mf/deref refs/workspace-project)
         file         (mf/deref workspace-file)
-        libraries    (mf/deref refs/workspace-libraries)
+        libraries    (d/filterm
+                       (fn [[key val]] (:is-direct val))
+                       (mf/deref refs/workspace-libraries))
```

Comment:
I think this is more legible using the "pipe" macro like:

```clojure
(->> (mf/deref refs/workspace-libraries)
     (d/filterm (fn [[key val]] (:is-direct val)))
```

Reply:
Done. I have also inverted the condition, because I want nils to pass the filter.